### DISCUSSION TITLE
1.7?

### DISCUSSION THREAD

**Discussion by hunte1r** [anirudhmalik/xhunter#55] [Q&A]
How is the progress in the development of the application and when will the next update 1.7 be released? i am really  Excited for it because the current application can not be used at all due to problems

**Comment by anirudhmalik** [comment]
Closing in favor of v2. See [v2.0-demo](https://github.com/anirudhmalik/xhunter/releases/tag/v2.0-demo) and the repo README. New bugs: open a fresh issue with repro steps. Thanks.
